ci/kokoro/docker/download-cache.sh

```diff
@@ -31,20 +38,30 @@ echo "max_size = 4.0G" >"${HOME_DIR}/.ccache/ccache.conf"
 readonly KEYFILE="${KOKORO_GFILE_DIR:-/dev/shm}/build-results-service-account.json"
 if [[ ! -f "${KEYFILE}" ]]; then
   echo "================================================================"
-  echo "Service account for cache access is not configured."
-  echo "No attempt will be made to download the cache, exit with success."
+  log_normal "Service account for cache access is not configured."
+  log_normal "No attempt will be made to download the cache, exit with success."
+  exit 0
+fi
+
+if [[ "${KOKORO_JOB_TYPE:-}" != "PRESUBMIT_GERRIT_ON_BORG" ]] && \
+   [[ "${KOKORO_JOB_TYPE:-}" != "PRESUBMIT_GITHUB" ]]; then
```

Comment:
Q. Is it worth just matching on `PRESUBMIT_.*`?

Reply:
I would prefer to not guess what the current and future job types mean.